src/components/HomePage/HomePage.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import ProductList from '../ProductList';
+import CartPageContainer from '../CartPage/CartPageContainer';
 
 const HomePage = () => (
   <div>
```

Comment:
React.Fragment can be used here instead of div or you can create class name for this component and add it to this div